server/schema/furniture.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE [IF NOT EXISTS] furniture (
+   id INT PRIMARY KEY,
```

Comment:
The `id` variable for both tables should be automatically generated using the `Serial` data type. Then, we also don't need to be passing in an ID when creating a new entry. Refer to this [article](https://www.geeksforgeeks.org/postgresql-serial/#:~:text=PostgreSQL%20has%20a%20special%20kind,Primary%20key%20of%20a%20table.&text=When%20creating%20a%20table%2C%20this,TABLE%20table_name(%20id%20SERIAL%20)%3B) for more about the serial data type.